lib/mod.lua

```diff
@@ -9,12 +10,25 @@ local my_midi = {
 function my_midi:send(data) end
 function my_midi:note_on(note, vel, ch)
     if ch == 1 then
+        if not channel_is_set_up then
+            print("Refusing to access nbout_chan_1")
+            return
+        end
         local p = params:lookup_param("nbout_chan_1"):get_player()
+        -- some sequencers (e.g. jala) seem to send nil for vel.
+        -- assume they want a default velocity
+        if vel == nil then
+            vel = 127
```

Comment:
Can we pick something more moderate, like 80 or 100 for default velocity?

Reply:
Yes, whatever you suggest.  I just chose 127 arbitrarily as something that I was sure to be able to hear (so that I could confirm that I was getting sound from jala).

I'll go with 80 for the moment.